hack/build.sh

```diff
@@ -23,9 +35,9 @@ function build_package() {
 }
 
 function build_all_backends() {
-  build_package backends/iptables
-  build_package backends/ipvs-as-sink
-  build_package backends/nft
+  cd ./backends
+  for f in $(find . -type f -name 'go.mod' | sed -r 's|/[^/]+$||' |sort |uniq)
```

Comment:
I think you can do a simpler `for f in $(find -name go.mod); do build_package $(dirname $f); done` (didn't test though ;))

Reply:
Ha! You are right. :)